doc/classes/EditorUndoRedoManager.xml

```diff
@@ -96,11 +96,13 @@
 			<param index="1" name="merge_mode" type="int" enum="UndoRedo.MergeMode" default="0" />
 			<param index="2" name="custom_context" type="Object" default="null" />
 			<param index="3" name="backward_undo_ops" type="bool" default="false" />
+			<param index="4" name="ignore_for_unsaved" type="bool" default="false" />
```

Comment:
Maybe `mark_unsaved` (flipping the bool) like used in the inspector sounds better?